src/main/java/com/example/bookstoreapp/service/BookService.java

```diff
@@ -12,7 +12,7 @@ public interface BookService {
 
     BookDto save(CreateBookRequestDto requestDto);
 
-    Page<BookDto> findAll(Pageable pageable);
+    Page<BookDto> findAll(String email, Pageable pageable);
```

Comment:
```suggestion
    Page<BookDto> findAll(Pageable pageable);
```
Why do you need this parameter?

Reply:
for Authentication by email specific user

Reply:
remove email